src/components/Todo.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import TodoForm from './TodoForm';
+import { RiCloseCircleLine } from 'react-icons/ri';
+import { TiEdit } from 'react-icons/ti';
+
+const Todo = ({ todos, completeTodo, removeTodo, updateTodo }) => {
```

Comment:
bonus points : @Abhishekcs194 can you figure out how to add eslint to the project? it should lint your code on precommit, you can learn it by googling it

Reply:
will do